### PR TITLE
Update authoring docs and readme for 1.1.4

### PR DESCRIPTION
**Pull request by angelazhangmsft**

### CODE REVIEW

File: docs/authoring.md [hunk 1]
```diff
@@ -115,9 +90,68 @@ Similarly, any other dependencies, e.g. `Microsoft.WinUI`, will need to be inclu
...
+
+Consuming a C#/WinRT component from a C++/WinRT desktop application is supported by both package reference or project reference.
+
+- For package references, simply right-click on the native project node and click **Manage NuGet packages** to find and install the component package.
+
+- For project references, you also currently need a reference to the component's generated WinMD. The WinMD can be found in the output directory of the authored component's project. To add both the project and WinMD references, right-click on the native project node, and click **Add** -> **Reference**. Select the C#/WinRT component project under the **Projects** node and the generated WinMD file from the **Browse** node.
+
+For native consumption of C#/WinRT components, you also need to create a manifest file named `YourNativeApp.exe.manifest`. If your app is packaged with MSIX, then you don't need to include the manifest file, otherwise you need to include your activatable class registrations in the manifest file. We provide an [authoring sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/AuthoringDemo/CppConsoleApp) with an example manifest file. To create the manifest file:
```
Comment (j0shuams) [inline docs/authoring.md:112]: "If your app is packaged with MSIX, then you don't need to create a manifest file. In the case that you do make a manifest, you need to add activatable class registrations for the public types in your component."

File: docs/authoring.md [hunk 2]
```diff
@@ -115,9 +90,68 @@ Similarly, any other dependencies, e.g. `Microsoft.WinUI`, will need to be inclu
...
+C#/WinRT supports authoring out-of-process components that can be consumed by other languages. Currently, consuming an out-of-process component is supported for managed C# apps with the use of a packaging project. A manual WinRT.Host.runtimeconfig.json file is currently required for this scenario, which is demonstrated in the sample below. Native consumption of out-of-process components is not fully supported yet.
+
+For an example of authoring and consuming an out-of-process C#/WinRT component, see the [background task component sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/BgTaskComponent).
+
+### Consuming from packaged applications
+
+Consuming C#/WinRT components from MSIX-packaged applications is supported for some scenarios. The latest Visual Studio Preview version is recommended for packaging scenarios.
+
+- Consuming a C#/WinRT component from packaged C# apps is supported.
+- Consuming a C#/WinRT component from packaged C++ apps is supported as a package reference, and requires the exe.manifest file.
```
Comment (j0shuams) [inline docs/authoring.md:135]: Maybe format exe.manifest -> `exe.manifest`

**Review by j0shuams — APPROVED** [review]
Awesome :D

**Comment by j0shuams** [comment]
@angelazhangmsft  Do C++ project references need to set Implementation on the winmd after the manual reference?

**Comment by angelazhangmsft** [comment]
> @angelazhangmsft Do C++ project references need to set Implementation on the winmd after the manual reference?

No this doesn't seem to be required. I think it's because of the `exe.manifest` registrations

**Pull request merged**